### PR TITLE
Characteristic read/write test.

### PR DESCRIPTION
**Pull request by cmumford**
A simple test to write to a characteristic value.

### CODE REVIEW

File: characteristic_readwrite/device_code.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    '1c84b596-222c-11eb-adc1-0242ac120002': {
+      '1c84b91a-222c-11eb-adc1-0242ac120002': {
+        value: data.buffer,
+        broadcast: false,
+        readable: true,
+        writable: true,
+        notify: false,
+        description: 'Single read-write characteristic',
+        onWrite: function(evt) {
+          data[0] = new DataView(evt.data).getUint32(0, /*littleEndian=*/ true);
```
Comment (reillyeon) [inline characteristic_readwrite/device_code.js]: nit: Here and throughout, remove the space before `true`.

File: characteristic_readwrite/web_app.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  });
+}
+
+function onGattDisconnected(evt) {
+  const device = evt.target;
+  logInfo(`Disconnected from GATT on device ${device.name}.`);
+}
+
+async function startTest() {
+  clearStatus();
```
Comment (reillyeon) [inline characteristic_readwrite/web_app.js:40]: I noticed while testing this manually that `clearStatus()` doesn't reset `numErrors` and so if a previous test run failed a successful run will still be marked as "FAIL".

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: characteristic_readwrite/device_code.js (hunk 1 above)
Comment (cmumford) [reply characteristic_readwrite/device_code.js]: clang-format is adding those. I'll look into what setting I can tweak to fix that. Manually removed for now.

File: characteristic_readwrite/web_app.js (hunk 2 above)
Comment (cmumford) [reply characteristic_readwrite/web_app.js:40]: Thx for catching that. Fixed.

**Comment by cmumford** [comment]
Thx for quick review. First set of changes made.

**Review by reillyeon — APPROVED** [review]

**Pull request merged**